specification/security/resource-manager/readme.nodejs.md

```diff
@@ -0,0 +1,14 @@
+## Node.js
+
+These settings apply only when `--nodejs` is specified on the command line.
+Please also specify `--node-sdks-folder=<path to root folder of your azure-sdk-for-node clone>`.
+
+``` yaml $(nodejs)
+nodejs:
+  azure-arm: true
+  package-name: azure-arm-security
+  output-folder: $(node-sdks-folder)/lib/services/SecurityManagement
```

Comment:
Please make `SecurityManagement` camel-cased (`securityManagement`).

Reply:
@daschult changed to `securityManagement`